packages/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -102,10 +109,29 @@ const TabBase: React.FunctionComponent<TabProps> = ({
 
   return (
     <li
-      className={css(styles.tabsItem, eventKey === localActiveKey && styles.modifiers.current, childClassName)}
+      className={css(
+        styles.tabsItem,
+        eventKey === localActiveKey && styles.modifiers.current,
+        handleTabClose && styles.modifiers.action,
+        childClassName
+      )}
       role="presentation"
     >
       {tooltip ? <Tooltip {...tooltip.props}>{tabButton}</Tooltip> : tabButton}
+      {handleTabClose !== undefined && (
+        <span className={css(styles.tabsItemClose)}>
+          <button
```

Comment:
Just curious - why not import and use `<Button>`?

Reply:
Ah you're right, I think was following the HTML a bit too closely. Will update that.